packit/config/package_config.py

```diff
@@ -254,6 +255,65 @@ def __eq__(self, other: object):
         )
 
 
+class PackageConfigValidation:
+    def __init__(self, package_config: PackageConfig):
+        from packit.schema import PackageConfigSchema
+
+        self.schema_validation_errors: Dict = {}
+        self.validation_errors: Dict = {}
```

Comment:
Are Dict okey or should I create class attribute for each validated field ?

Reply:
Dict is just fine I'd say